package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@jupiterone/graph-hubspot",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "A JupiterOne Integration",
   "license": "MPL-2.0",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "src/index.js",
+  "types": "src/index.d.ts",
   "files": [
-    "dist"
+    "src"
```

Comment:
any questions in the /jupiterone dir?

Reply:
graph-github specifies jupiterone here..

Reply:
Crap.  Good catch.  That'll bite me shortly if I don't add a questions file.  Thanks!